themesource/datagrid/web/datagrid-filters.scss

```diff
@@ -94,6 +94,81 @@ $arrow: "data:image/svg+xml;utf8,<svg xmlns='http://www.w3.org/2000/svg' width='
     }
 }
 
+.filter-selectors {
```

Comment:
I noticed the outline styles being cut off at the rounded corners of the filter menu. Is that issue introduced with these changes? Or was that already the case?

Reply:
I will leave this one for future improvement as it only happens in Chrome